Added February community call summary post (#395)
USRSE/usrse.github.io: _posts/2021-02-19-community-call-education.md

---
layout: post
title: February Community Call Summary

> tags: [community call]

Reviewer: ```suggestion
tags: [community call, education, training]
```

posted_by: Julia Damerow, Chris Hill
---

> The February US-RSE Community Call focused on the topic of RSE education and training. It was well attended with about 25 participants. After some updates from the steering committee we started out with a few warm up Mentimeter questions. Turns out Star Wars created a lot of favorite fictional characters (surprise ;) ). It got interesting with the question if there should be some kind of formal RSE training. While 12 people thought there should be, 11 people answered that it depends, while one person didn’t think there should be formal training. This might warrant a follow up discussion on the question of under what circumstances a formal RSE training would be useful or even desired.

Reviewer 2: ```suggestion
The February US-RSE Community Call focused on the topic of RSE education and training. It was well attended with about 25 participants. After some updates from the steering committee we started out with a few warm up Mentimeter questions. Turns out Star Wars created a lot of favorite fictional characters (surprise ;) ). It got interesting with the question if there should be some kind of formal RSE training. While 12 people thought there should be, 11 people answered that it depends, while one person didn’t think there should be formal training. This might warrant a follow-up discussion on the question of under what circumstances a formal RSE training would be useful or even desired.
```

![Do you think there should be some kind of formal training for RSEs?]({{ site.baseurl }}/assets/posts/2021-02-19-community-call-education.png "Do you think there should be some kind of formal training for RSEs?")

> We next asked all participants what the last RSE skill that they learnt was. Interestingly, many people listed DevOps skills such as Docker, Kubernetes, CI/CD, or GitHub Actions, which might be an indicator that many RSE jobs not just require programming skills but also expertise in infrastructure-related topics. We ended the Mentimeter questions by asking what skill or technology was highest on the participants’ list to learn. While several participants responded with programming related skills such as learning Rust, Python, or Machine Learning, there were also topics such as CI/CD, career management, and project management.

Reviewer 2: ```suggestion
We next asked all participants what the last RSE skill that they learned was. Interestingly, many people listed DevOps skills such as Docker, Kubernetes, CI/CD, or GitHub Actions, which might be an indicator that many RSE jobs not just require programming skills but also expertise in infrastructure-related topics. We ended the Mentimeter questions by asking what skill or technology was highest on the participants’ list to learn. While several participants responded with programming related skills such as learning Rust, Python, or Machine Learning, there were also topics such as CI/CD, career management, and project management.
```

> After the Mentimeter sessions, Jeffrey Carver talked about the Intersect Project run by him and Ian Cosden. The Intersect Project will create training materials, host workshops, and bootcamps for RSEs. More information can be found [here](https://intersect-training.github.io/). Jeffrey’s presentation was followed by Nick Murphy, who talked about his work with the plasma science community on best programming practices and open source (his slides can be [here](https://zenodo.org/record/3922957#.YC_ujy1h3Ck)).

Reviewer 2: ```suggestion
After the Mentimeter sessions, Jeffrey Carver talked about the Intersect Project that he and Ian Cosden run. The Intersect Project will create training materials, host workshops, and bootcamps for RSEs. More information can be found [here](https://intersect-training.github.io/). Jeffrey’s presentation was followed by Nick Murphy, who talked about his work with the plasma science community on best programming practices and open source (his slides can be [here](https://zenodo.org/record/3922957#.YC_ujy1h3Ck)).
```

These two very interesting presentations were followed by breakout sessions in which the participants discussed the following questions.
- How did you obtain your software engineering skills?
- Have you ever trained other RSEs?
- How do you believe the type of training an RSE receives influences their work?
- Are there areas of technical or non-technical knowledge where training is not easily available to you at the moment?

> Generally, there seemed to be a mix of people having some kind of formal computer science training and people who learnt their RSE skills on the job. While several participants had some experience with training, a lot of that experience was through mentoring or teaching classes for undergraduate students, graduate students, or postdocs. The [Software Carpentry](https://software-carpentry.org/) was mentioned as a good resource for training.

Reviewer 2: ```suggestion
Generally, there seemed to be a mix of people having some kind of formal computer science training and people who learnt their RSE skills on the job. While several participants had some experience with training, a lot of that experience was through mentoring or teaching classes for undergraduate students, graduate students, or postdocs. [Software Carpentry](https://software-carpentry.org/) was mentioned as a good resource for training.
```

> Regarding the question of how the type of training of an RSE influences their work a couple of interesting points were made. One participant suggested that the recognition of the work of an RSE can depend on how close they are to the subject area of a project. Someone else pointed out that the choices an RSE makes are limited by the knowledge and experiences of that RSE, since you can’t choose something that you don’t know of. Therefore, it’s important to keep learning about new technologies and to stay up-to-date with current developments in the field.

Reviewer 2: ```suggestion
Regarding the question of how the type of training of an RSE influences their work, a couple of interesting points were made. One participant suggested that the recognition of the work of an RSE can depend on how close they are to the subject area of a project. Someone else pointed out that the choices an RSE makes are limited by the knowledge and experiences of that RSE, since you can’t choose something that you don’t know of. Therefore, it’s important to keep learning about new technologies and to stay up-to-date with current developments in the field.
```
